pages/linux/pacman-database.md

```diff
@@ -0,0 +1,28 @@
+# pacman --database
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --database --help`
+
+- Mark a package as non-explicitly installed:
```

Comment:
What do you think about 'implicitly'?  Or does that deviate too much from the official terminology?
```suggestion
- Mark a package as implicitly installed:
```

Reply:
maybe we can use `dependency for another package`

Reply:
Nah&mdash;I think the original is fine.  I wasn't advocating _against yours_, I was **just** advocating _for mine._ 🙂

Reply:
`implicitly` is the correct grammatical term to use here.